### PR TITLE
Update copy right, remove broucher, case study and fix contact icons

### PR DESCRIPTION
**Pull request by mohammadalihera**

### CODE REVIEW

File: account.html
Comment (sayam56) [inline account.html:1]: What is this file and what are the changes on it?

File: application_case_study.html
Comment (sayam56) [inline application_case_study.html:1]: Why do we have changes in case study too?

File: faq.html [hunk 1]
```diff
@@ -579,7 +578,7 @@ <h4>Contact info</h4>
 
           <!-- Footer Bottom -->
           <div class="footer-bottom">
-            <div class="copyright">Copyright © 2021 NerdEvolution. All Rights Reserved.</div>
+            <div class="copyright">Copyright © 2024NerdEvolution. All Rights Reserved.</div>
```
Comment (sayam56) [inline faq.html:581]: Space Missing

File: brand_identity.html [hunk 2]
```diff
@@ -479,7 +478,7 @@ <h4>Contact info</h4>
 
           <!-- Footer Bottom -->
           <div class="footer-bottom">
-            <div class="copyright">Copyright © 2021 NerdEvolution. All Rights Reserved.</div>
+            <div class="copyright">Copyright © 2024NerdEvolution. All Rights Reserved.</div>
```
Comment (sayam56) [inline brand_identity.html:481]: Space missing in all of them

File: functional-resume.html [hunk 3]
```diff
@@ -487,7 +486,7 @@ <h4>Contact info</h4>
 
           <!-- Footer Bottom -->
           <div class="footer-bottom">
-            <div class="copyright">Copyright © 2021 NerdEvolution. All Rights Reserved.</div>
+            <div class="copyright">Copyright © 2024NerdEvolution. All Rights Reserved.</div>
```
Comment (sayam56) [inline functional-resume.html:489]: Please fix this spacing issue in all of them

File: not-found.html
Comment (sayam56) [inline not-found.html:1]: Why changes in here?

File: ui_ux_detail.html
Comment (sayam56) [inline ui_ux_detail.html:1]: Are these new contents from the team? I am a bit confused

**Review by sayam56 — CHANGES_REQUESTED** [review]
Please check the comments and update/ provide feedback as needed

File: faq.html (hunk 1 above)
Comment (mohammadalihera) [reply faq.html:581]: Fix review

File: account.html
Comment (mohammadalihera) [reply account.html:1]: Here I change only in the copy right section

File: application_case_study.html
Comment (mohammadalihera) [reply application_case_study.html:1]: I have changed the copy right by searching it and replacing it. so it became change everywhere

File: functional-resume.html (hunk 3 above)
Comment (mohammadalihera) [reply functional-resume.html]: Fixed this spacing issue in all of the pages

File: account.html
Comment (sayam56) [reply account.html:1]: image
But look at the diffs, there's so many changes. Did you pull from master before making the changes and submitting the PR?

File: application_case_study.html
Comment (sayam56) [reply application_case_study.html:1]: image
Again, I see a lot of changes other than just the copyright information. Did you make sure to pull from main/'master before submitting the PR?

File: not-found.html
Comment (sayam56) [reply not-found.html:1]: image
I see a lot more than just copyright changes. Please let me know what these are.

File: ui_ux_detail.html
Comment (sayam56) [reply ui_ux_detail.html:1]: @mohammadalihera Please justify this as well. Thanks

**Review by sayam56 — CHANGES_REQUESTED** [review]

File: ui_ux_detail.html
Comment (mohammadalihera) [reply ui_ux_detail.html:1]: Yes, Ui Ux is new contents, but I need the images and content for this screen

Comment (sayam56) [reply ui_ux_detail.html:1]: If this screen is not ready to be public yet, please make sure you remove the links to this page from the UI.

File: account.html
Comment (mohammadalihera) [reply account.html:1]: image
Yes I took pull. and in the git hub I am not seeing these chnages, is your diff upto date?

**Pull request merged**